conf/base_config.yaml

```diff
@@ -32,10 +33,10 @@ simulator:
         n_bands: 1
         batch_size: 64
         max_sources: 1
-        mean_sources: 0.2
+        mean_sources: 0.03
         min_sources: 0
         prob_galaxy: 0.72
-        star_flux_min: 622.0
+        star_flux_min: 1e5
```

Comment:
622 is more realistic. Can you override this constant in a case study-specific config file rather than changing it throughout?

Reply:
Changed this accidentally, reverted to previous value.